apps/artist/client/cta.coffee

```diff
@@ -1,36 +1,37 @@
-{ REFERRER, APP_URL, CURRENT_USER, SHOW_ARTIST_CTA } = require('sharify').data
```

Comment:
This file, we basically early return if it's an internal Artsy browsed-to artist page, or if you're logged in.

Otherwise, we either show you the new or old CTA bar, that the split test sets for us.